packages/app-bridge/src/react/useGuidelineActions.ts

```diff
@@ -505,15 +509,15 @@ export const useGuidelineActions = (appBridge: AppBridgeTheme) => {
 
     const moveDocumentPageBetweenDocuments = useCallback(
         async (documentPageId: number, sourceDocumentId: number, targetDocumentId: number) => {
-            await appBridge.moveDocumentPageBetweenDocuments(documentPageId, targetDocumentId);
+            const result = await appBridge.moveDocumentPageBetweenDocuments(documentPageId, targetDocumentId);
 
-            window.emitter.emit(`AppBridge:GuidelineDocumentPageAction:${sourceDocumentId}`, {
-                documentPage: { id: documentPageId, documentId: sourceDocumentId } as DocumentPage,
-                action: 'update',
+            window.emitter.emit('AppBridge:GuidelineDocumentPageAction', {
+                documentPage: { id: documentPageId, documentId: sourceDocumentId, categoryId: null },
+                action: 'delete',
```

Comment:
Why are we doing delete and update actions? I feel this is wrong.

We have the following scenarios:

1. page is in a category in 1st document and it's being moved to 2nd document uncategorized.
2. page is uncategorized in 1st document and it's being moved to 2nd document in the category.
3. page is uncategorized in 1st document and it's being moved to 2nd document uncategorized.
4. page is in a category in 1st document and it's being moved to 2nd document in the category. 

For the 1st two scenarios, we need to do the 'delete' and 'add' actions, and for 2nd two scenarios, we need to do only the update action.

Correct me if I'm wrong or missing something.